src/filename.jsx

```diff
@@ -23,7 +30,8 @@ class Filename extends Component {
 
     render() {
         return (
-            <span className={ `react-fine-uploader-filename ${this.props.className || ''}` }>
+            <span className={ `react-fine-uploader-filename ${this.props.className || ''}` }
+                  onClick={this._onClick}>
```

Comment:
Why does the `<Filename>` component have a click handler? Furthermore, this is completely inaccessible.